Packs/Threat_Crowd/Integrations/ThreatCrowdV2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,270 @@
+        res['value'] = antivirus
+
+        markdown = tableToMarkdown(f"Threat crowd report for antivirus {antivirus}", res)
+
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.AntiVirus',
+            outputs=res.copy(),
+            outputs_key_field='value',
+            readable_output=markdown,
+            raw_response=res
+        ))
+
+    return command_results
+
+
+def file_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'file/report/'
+    files = argToList(args.get('file'))
+    for file_hash in files:
+        res = client.http_request(url_postfix=api_url, **{'resource': file_hash})
+        res['value'] = file_hash
+
+        score = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            file_hash, DBotScoreType.FILE, VENDOR, score, reliability=client.reliability)
+        file_object = Common.File(dbot, md5=res.get('md5'), sha1=res.get('sha1'))
+
+        markdown = tableToMarkdown(f"Threat crowd report for File {file_hash}", res)
+
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.File',
+            outputs=res.copy(),
+            outputs_key_field='value',
+            indicator=file_object,
+            readable_output=markdown,
+            raw_response=res
+        ))
+
+    return command_results
+
+
+def test_module(client: Client) -> str:
+    try:
+        api_url = 'ip/report/'
+        client.http_request(url_postfix=api_url, **{'ip': '1.1.1.1'})
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    command_functions = {'email': email_command,
+                         'domain': domain_command,
+                         'ip': ip_command,
+                         'threat-crowd-antivirus': antivirus_command,
+                         'file': file_command}
+
+    params = demisto.params()
+    base_url = params.get('server_url')
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    reliability = params.get('integrationReliability')
+    reliability = reliability if reliability else DBotScoreReliability.C
+
+    if DBotScoreReliability.is_valid_type(reliability):
+        reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+    else:
+        raise Exception("Please provide a valid value for the Source Reliability parameter.")
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+
+        client = Client(
```

Comment:
I think we should tend to the `extended_results` params in the client.
assign it here. use it in funcs if it is true, else limit to the default decided on with the PM.